packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -416,6 +415,18 @@ def get(self):
     required=False,
     default=False,
 )
+# todo: remove after clients are updated to use /download
+stream_parser.add_argument(
+    "filename",
+    description="""Optional - name of file to download. Redirects to /download if provided.""",
```

Comment:
Let's prefix both of these descriptions with "DEPRECATED" - I wonder if we can get the SDK gen to include the `@deprecated` somehow